src/snippets/snippets.ts

```diff
@@ -50,6 +57,7 @@ export class UIPSnippets extends UIPPlugin {
     return $scroll;
   }
 
+  /** Render snippets list. */
   protected rerender(): void {
```

Comment:
It's just look strange for me to see comment with _render_ to method called _rerender_. At all, i'd better rename this method, cause by it's name i have no idea what it does and what it rerenders